allow fixed width option for custom table cell status card

sijanstu opened the pull request:
I have added optional fixed width for status, please link the issue and review. Thanks

sijanstu committed:
allow fixed width option for custom table cell status card

sijanstu commented:
Proposed UI should look like this:
<img width="1367" alt="Screenshot 2025-06-12 at 4 34 25 PM" src="https://github.com/user-attachments/assets/da9c7a39-aca6-4aa8-89b6-184a58fedb3d" />

DJ-Raven commented on custom-table-cell/src/main/java/raven/table/TableBadgeCellRenderer.java:
```diff
@@ -13,11 +13,26 @@ public class TableBadgeCellRenderer extends JLabel implements TableCellRenderer
+            setVerticalAlignment(CENTER);
+            setHorizontalTextPosition(RIGHT);
+            setVerticalTextPosition(CENTER);
+            setIconTextGap(UIScale.scale(5));
```
For IconTextGap should set by use flatlaf style properties. so it can be scale at runtime

DJ-Raven replied:
can be add to this:

https://github.com/DJ-Raven/raven-java-swing-tutorial-project/blob/af1b473209e5f1c9098f9a21eb0487cffa9a8530/custom-table-cell/src/main/java/raven/table/TableBadgeCellRenderer.java#L19-L21

`iconTextGap:5`

sijanstu replied:
I have made changes according to your suggestions.

sijanstu committed:
Use FlatLaf style properties for iconTextGap in TableBadgeCellRenderer

- Added iconTextGap to the FlatLaf style properties to support proper scaling at runtime
- Removed direct call to setIconTextGap(UIScale.scale(5))
- Simplified styling by combining border and iconTextGap in a single style property

DJ-Raven commented:
@sijanstu Thank you 😀
